### PR TITLE
Move annotations to separated yml files

### PR DESCRIPTION
**Pull request by vhr**
# Pull Request Template

## Description

The purpose of the PR is to move all annotations in src in YAML configuration files or other way of describing

**Fixes** MON-4795

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x
- [ ] 19.10.x
- [x] 20.04.x (master)

<h2> How this pull request can be tested ? </h2>

the responses from APIv2 have to be the same as before

#### Community contributors & Centreon team

- [x] I followed the **coding style guidelines** provided by Centreon
- [x] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [x] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have made corresponding changes to the **documentation**.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

#### Centreon team only

- [ ] I have made sure that the **unit tests** related to the story are successful.
- [ ] I have made sure that **unit tests cover 80%** of the code written for the story.
- [x] I have made sure that **acceptance tests** related to the story are successful (**local and CI**)

### CODE REVIEW

File: templates/base.html.twig [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```
Comment (kduret) [inline templates/base.html.twig]: is this file needed ?

File: config/packages/serializer/Centreon/Acknowledgement.Acknowledgement.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+Centreon\Domain\Acknowledgement\Acknowledgement:
```
Comment (kduret) [inline config/packages/serializer/Centreon/Acknowledgement.Acknowledgement.yml]: Could you move all serializer files to "yaml" extension (instead of yml) to let it consistent with other

Comment (vhr) [reply config/packages/serializer/Centreon/Acknowledgement.Acknowledgement.yml]: it's a requirement by the bundle

> If you are using YAML files as metadata format, the file extension to use is .yml

https://jmsyst.com/bundles/JMSSerializerBundle/master/configuration#defining-metadata

File: templates/base.html.twig (hunk 1 above)
Comment (vhr) [reply templates/base.html.twig]: in dev mode yes it's needed because the WebProfilerBundle use twig as a template engine

Comment (kduret) [reply templates/base.html.twig]: WebProfilerBundle does not use this one ? https://github.com/symfony/web-profiler-bundle/blob/master/Resources/views/Profiler/base.html.twig

Comment (vhr) [reply templates/base.html.twig]: yes you are right, there is no need from that file
